tailwind.config.mjs

```diff
@@ -34,6 +44,35 @@ export default {
         base: ['13px', '1.4'],
         lg: ['14px'],
         xl: ['16px'],
+        annotator: ['12px'],
```

Comment:
Unlike the other variants here it isn't really obvious that this refers to a size when used as `text-annotator`. I don't have a particular alternative in mind.

Reply:
I would suggest moving the line between `sm` and `base`. Maybe the name `base-annotator`?

Reply:
I'll rename and move. Thanks for the feedback, both of you.